modules/admin_manual/pages/configuration/user/login_policies.adoc

```diff
@@ -0,0 +1,64 @@
+= Login Policies
+:toc: right
+
+:description: ownCloud provides login policies so admins can allow or reject groups of users to access ownCloud via specific login mechanisms.
+
+== Introduction
+
+{description}
+
+Login policies will emit a `failed login` event if the user isn't allowed to login due to a reject rule.
```

Comment:
I don't know if this is clear enough. I mean, taking into account only the current group policy, you can set just a list of "allowOnly" groups, allowing only users in those groups. In that case, there is no explicit reject rule.